pkg/domain/entities/pods.go

```diff
@@ -178,6 +190,14 @@ func (p *PodCreateOptions) ToPodSpecGen(s *specgen.PodSpecGenerator) error {
 	s.NoManageHosts = p.Net.NoHosts
 	s.HostAdd = p.Net.AddHosts
 
+	pidR, err := setNamespaces(p)
```

Comment:
Minor nit, but slightly more descriptive variable names would be nice - I don't really like having a function with `pid` and `pidR`